compute_tools/src/spec_apply.rs

```diff
@@ -697,6 +700,39 @@ async fn get_operations<'a>(
             }
             Ok(Box::new(empty()))
         }
+        ApplySpecPhase::CreatePgauditExtension => {
+            if let Some(libs) = spec.cluster.settings.find("shared_preload_libraries") {
```

Comment:
Should we just mix-in all required extensions into preload libs if audit is enabled? It's not convenient and seems bug-prone if the caller (control plane) need to pass several toggles in a right combination to make it working

I can understand passing some extra parameters/overrides, but here it looks like essential thing for oeprations

Reply:
>Should we just mix-in all required extensions into preload libs if audit is enabled?

yes, actually, I already do it in cloud code.
I'll remove this check, because it is redundant.

Reply:
> yes, actually, I already do it in cloud code.

I think it should be done in compute_ctl code because 

> It's not convenient and seems bug-prone if the caller (control plane) need to pass several toggles in a right combination to make it working

Reply:
Fixed